catalyst_voices/packages/catalyst_voices_brands/lib/src/themes/catalyst.dart

```diff
@@ -1,10 +1,231 @@
 import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_assets/generated/colors.gen.dart';
+import 'package:catalyst_voices_brands/src/themes/extended_color_scheme.dart';
 import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+ThemeData _buildThemeData(
+  ColorScheme colorScheme,
+  ExtendedColorScheme extendedColorScheme,
+) {
+  return ThemeData(
+    textTheme: TextTheme(
+      displayLarge: GoogleFonts.notoSans(
+        color: extendedColorScheme.textPrimary,
+        fontSize: 57,
+        letterSpacing: -1.14,
+        fontWeight: FontWeight.w700,
+        height: -0.02,
+      ),
+      displayMedium: GoogleFonts.poppins(
+        color: extendedColorScheme.textPrimary,
+        fontSize: 45,
+        fontWeight: FontWeight.w700,
+        height: 0.03,
+      ),
+      displaySmall: GoogleFonts.poppins(
+        color: extendedColorScheme.textPrimary,
+        fontSize: 36,
+        fontWeight: FontWeight.w700,
+        height: 0.03,
+      ),
+      headlineLarge: GoogleFonts.poppins(
+        color: extendedColorScheme.textPrimary,
+        fontSize: 32,
+        fontWeight: FontWeight.w700,
+        height: 0.04,
+      ),
+      headlineMedium: GoogleFonts.poppins(
+        color: extendedColorScheme.textPrimary,
+        fontSize: 28,
+        fontWeight: FontWeight.w700,
+        height: 0.05,
+      ),
+      headlineSmall: GoogleFonts.poppins(
+        color: extendedColorScheme.textPrimary,
+        fontSize: 24,
+        fontWeight: FontWeight.w700,
+        height: 0.06,
```

Comment:
From the docs:

```
The height of this text span, as a multiple of the font size.
```

This property is used to lay out the text in multiple lines, the bigger you make it the more space will be between lines in a text block. 0.06 seems extremely small 🤔

Reply:
![Screenshot 2024-05-09 at 12 50 00](https://github.com/input-output-hk/catalyst-voices/assets/166132265/d6dcb9e7-db74-4b46-839f-a9c810773b44)

I usually do it like this:
height = line height from figma / font size

Reply:
I used the code generator in Figma to obtain these values. I guess it's not working as it should :/